src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -3421,28 +3421,28 @@ let AnalyzeArbitraryExprAsEnumerable (cenv: cenv) (env: TcEnv) localAlloc m expr
     let tryType (exprToSearchForGetEnumeratorAndItem, tyToSearchForGetEnumeratorAndItem) =
         match findMethInfo true m "GetEnumerator" tyToSearchForGetEnumeratorAndItem with
         | Exception exn -> Exception exn
-        | Result getEnumerator_minfo ->
+        | Result getEnumeratorMethInfo ->
```

Comment:
Maybe "Method"? Or do we already have "meth" all around the code base? :)

Reply:
`MethInfo`/`PropInfo`  are standard type names in `infos.fsi`.

`MethodInfo` and `PropertyInfo` don't really fly because they can be confused with the `System.Reflection` types.

Reply:
Sure, let it be then